app/views/layouts/mailer.html.erb

```diff
@@ -3,11 +3,41 @@
   <head>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <style>
-      /* Email styles need to be inline */
+      * {
+        font-family: Helvetica Nue, Helvetica, Arial, sans-serif;
+      }
+      .wrapper {
+        max-width: 500px;
+        margin: 0 auto;
+      }
+      .wrapper p {
+        margin: 1.5rem 0;
+      }
+      .logo {
+        margin: 2.5rem 0;
+      }
+      .logo img {
+        width: 180px;
+        height: auto;
+      }
+      .subject {
+        font-size: 20px;
+        color: #757575;
+        font-weight: normal;
+        margin-bottom: 2.5rem;
+      }
     </style>
   </head>
 
   <body>
-    <%= yield %>
+    <div class="wrapper">
+      <div class="logo">
+        <%= image_tag "GYR_logo.png", style: "width:180px;height:auto" %>
```

Comment:
Does this image tag involve a `<img src="https://getyourrefund.org/...">`? If so, then email users with email image privacy features enabled won't see the image. That's OK, but honestly I love email image privacy features, and personally want everyone to enable them, and have them enabled myself, and would like to see GYR emails w/ the logo.

To get the logo into the message itself, you can use a reference like this: http://www.lindsaar.net/2010/6/7/inline-attachments-for-actionmailer

Not a blocker, but I love it, so there you go.

Reply:
Blocker: When I use this code on my laptop, I see the IMG tag has a `src="/assets/...` path, which is not an absolute URL. This will mean it will look like it works in the preview you saw, but actually never show an image in a client's email client. If you stick to using the URL to our site (which is OK), make sure you use an absolute URL not a path.

Reply:
switched to the inlined attachment, but also set it up so that we're using asset_host for emails in config for demo, staging, and prod.